pet.el

```diff
@@ -1003,6 +1004,7 @@ buffer local values."
   (setq-local lsp-jedi-executable-command
               (pet-executable-find "jedi-language-server"))
   (setq-local lsp-pyls-plugins-jedi-environment python-shell-virtualenv-root)
+  (setq-local lsp-pylsp-server-command (list (pet-executable-find "pylsp")))
```

Comment:
Why do you need to find this from your project's virtualenv? What kind of setup and configuration is required for this to work?